README.md

```diff
@@ -88,6 +88,14 @@ sudo dnf install gtk3-devel webkit2gtk4.1-devel fuse3-devel
 sudo pacman -S webkit2gtk-4.1 fuse3
 ```
 
+## Installation
+Clone this repo to a location of your choosing. Then
+```
+git submodule update && git submodule init
+```
+to use the patched version of `ureq`. You can then `cargo run -p bbfs-cli` or `cargo install --bin bbfs-cli`
```

Comment:
We probably don't need to mention that the submodule commands are for ureq because that's beside the point. I think it'd probably just confuse less rusty people who are trying to install the package.

Reply:
Yep definitely, I've changed this!